ipatests/test_integration/test_sssd.py

```diff
@@ -108,3 +113,45 @@ def test_auth_cache_enabled_when_configured(self, user):
             assert self.is_auth_cached(self.users[user])
             time.sleep(10)
             assert not self.is_auth_cached(self.users[user])
+
+    @contextmanager
+    def filter_user_setup(self, user):
+        sssd_conf_backup = tasks.FileBackup(self.master, paths.SSSD_CONF)
+        filter_user = {'filter_users': '{0}'.format(self.users[user]['name'])}
```

Comment:
`'{0}'.format(self.users[user]['name'])` can be replaced with `self.users[user]['name']`